source/fundamentals/indexes.txt

```diff
@@ -426,31 +424,28 @@ in the {+mdb-server+} manual.
 Unique Indexes
 ~~~~~~~~~~~~~~
 
-.. _unique-indexes:
-
 Unique indexes ensure that the indexed fields do not store duplicate values. By default, MongoDB creates a unique index
 on the ``_id`` field during the creation of a collection. To create a unique index, specify the field or combination of
 fields that you want to prevent duplication on and set the ``unique`` option to ``true``.
 
 The following example creates a unique, descending index on the ``theaterId`` field:
 
-.. literalinclude:: /includes/fundamentals/code-snippets/IndexPage.java
-   :language: java
+.. literalinclude:: 
+   :language: csharp
```

Comment:
I: This code block is no longer rendering on the staging site - I think you can just change it back to how it was

Reply:
Not sure why so many unrelated changes/modifications/deletions got included in this commit, let me investigate to make sure I pulled in the changes correctly